src/main/java/com/amazon/dlic/auth/http/jwt/AbstractHTTPJwtAuthenticator.java

```diff
@@ -120,7 +120,7 @@ private AuthCredentials extractCredentials0(final RestRequest request) throws El
 
         if (subject == null) {
             log.error("No subject found in JWT token");
-            return null;
+            throw new ElasticsearchSecurityException("No subject found in JWT token", RestStatus.FORBIDDEN);
```

Comment:
Do we need tests for this change?

Reply:
Added a unit test called "wrongSubjectKeyTest" below

Reply:
Should we verify that the error gets thrown out in the test?